docs/addons/xray.md

```diff
@@ -22,11 +22,11 @@ const blueprint = blueprints.EksBlueprint.builder()
   .build(app, 'my-stack-name');
 ```
 
-Once deployed, it allows applications to be instrumented with X-Ray by leveraging the X-Ray SDK.  Examples of such integration can be found on [GitHub](https://github.com/aws-samples/aws-xray-kubernetes).
+Once deployed, it allows applications to be instrumented with X-Ray by leveraging the X-Ray SDK.  Examples of such integration can be found on [AWS Docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy_servicelens_CloudWatch_agent_deploy_EKS.html).
 
 ## Functionality
 
-1. Creates the `xray-system` namespace.
-2. Deploys the [`xray-daemon`](https://www.eksworkshop.com/intermediate/245_x-ray/x-ray-daemon/) manifests into the cluster.
-3. Configures Kubernetes service account with IRSA (`AWSXRayDaemonWriteAccess`) for communication between the cluster and the AWS X-Ray service 
+1. Creates the `amazon-cloudwatch` namespace.
+2. Deploys the [`xray-daemon`](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy_servicelens_CloudWatch_agent_deploy_EKS.html) manifests into the cluster.
+3. Configures Node role with policies (`CloudWatchAgentServerPolicy`, `AWSXRayDaemonWriteAccess`) for communication between the cluster and the CloudWatch and X-Ray service.
```

Comment:
It looks like this setup deploys CW agent and XRay into the cluster. Why would we do that? Can we drop the agent deployment and just deploy XRay? I also assume that CloudWatchAgentServerPolicy won't be needed if we only deploy xray.